README.md

```diff
@@ -62,6 +63,51 @@ git clone https://github.com/apache/lucene.git lucene_baseline
 
 Adjust the command accordingly for `lucene_candidate`.
 
+Compile and build the lucene_baseline and lucene_candidate projects as per its README.txt
+
+If you checkout Lucene in a different directory, update the same in the build.properties to use the right lucene jars in
+this project for compilation. 
+
+### (Optional, for development) set up IntelliJ
+You can open the project as usual in IntelliJ. 
+
+Select src/main folder as Source Root. `Right Click src/main --> Mark Directory As --> Source Root`
+
+Select src/python folder as source Root. `Right Click src/python --> Mark Directory As --> Source Root`
+
+For configuring the right jar files you will have to do the following.
+The jars that you need are present in at 3 places.
+1. Apache Lucene jars. Which are present in your ${lucene.checkout}/lucene folder. 
+2. Some jars will be downloaded by Ant-Ivy build system of this project into the `./libs` folder of this package.
+3. Some jars are shipped with this project which are present in the `./lib` folder.
+
+IntelliJ Idea does not automatically detect these jars in the classpath. So you will have to do following steps to 
+include them. Also, IntelliJ Idea by default do not recursively include all the jars in sub folders, so follow the below
+instructions to enable recursive search in folder for both the above jar paths.
+
+``
+IntelliJ IDEA --> File --> Project Structure --> Modules --> luceneutil-root --> Dependencies --> + 
+--> Jars And Directories -->  Select the path to ${lucene.checkout}/lucene --> Open --> Choose Roots (Unselect All) --> 
+OK --> Choose Categories of Selected Files --> Jar Directory --> OK --> Apply --> OK
+``
+
+After this you will have to update the file in .idea folder manually to enable 'recursive' jar discovery. Open your 
+project's `.iml` file and change the `<jarDirectory>` xml tag's recursive attribute to `true`
+` vim $luceneutil.root/.idea/luceneutil.iml `
+
+Change line like below from recursive="false" to recursive="true"
+``` xml
+<library>
+  ...
+  <jarDirectory url="file://$MODULE_DIR$/../../lucene/lucene" recursive="false" />
+</library>
+  ```
+
+This will make idea search jar files recursively in lucene project.
+Alternatively, you can configure idea with the source code of Lucene as a library, by not doing "unselecting all" in the `Choose Roots` step above.
+
+Repeat the same process for lib and libs folder in $luceneutil.root folder.
+
```

Comment:
Agree, IDEs understand Gradle much more than Ant. But I also felt that there is a learning curve for gradle, especiallly because of language dependence. Also, Although there is java toolkit support in Gradle now, but there can be some difficulties in supporting newer or custom JDK for testing. But such does not seem to be an issue with Ant as it does not come in between execution of code. 
With this small initial setup we can work with both lucene classes or lucene jars. Moreover if all the lucene jars are put in one folder in its build then we don't need to manually add recursive in *.iml file.